blogs/tests/test_parser.py

```diff
@@ -15,17 +14,13 @@ def setUpClass(cls):
 
     def test_entries(self):
         self.assertEqual(len(self.entries), 25)
-        self.assertEqual(
-            self.entries[0]['title'],
-            'Introducing Electronic Contributor Agreements'
-        )
+        self.assertEqual(self.entries[0]["title"], "Introducing Electronic Contributor Agreements")
         self.assertIn(
-            "We're happy to announce the new way to file a contributor "
-            "agreement: on the web at",
-            self.entries[0]['summary']
+            "We're happy to announce the new way to file a contributor " "agreement: on the web at",
```

Comment:
Turn on ISC to catch these:

```suggestion
            "We're happy to announce the new way to file a contributor agreement: on the web at",
```

https://docs.astral.sh/ruff/rules/#flake8-implicit-str-concat-isc